src/main/java/com/android/volley/toolbox/DiskBasedCacheUtility.java

```diff
@@ -37,6 +43,95 @@ static String getFilenameForKey(String key) {
         return localFilename;
     }
 
+    /** Returns a file object for the given cache key. */
+    public static File getFileForKey(String key, FileSupplier rootDirectorySupplier) {
+        return new File(rootDirectorySupplier.get(), getFilenameForKey(key));
+    }
+
+    static boolean checkPrune(long totalSize, int dataLength, int maxCacheSize) {
+        return (totalSize + dataLength > maxCacheSize);
+    }
+
+    static boolean checkWouldDelete(int dataLength, int maxCacheSize) {
```

Comment:
Could you find a more descriptive name here as well?

Reply:
changed